packages/terra-action-header/src/ActionHeader.jsx

```diff
@@ -75,6 +89,10 @@ const defaultProps = {
   onNext: undefined,
   onPrevious: undefined,
   children: undefined,
+  backButtonA11yLabel: undefined,
```

Comment:
Why must a required prop specify a default value?

Reply:
It is not a required prop.